Tools/WebKitTestRunner/cocoa/TestWebsiteDataStoreDelegate.mm

```diff
@@ -153,6 +153,14 @@ - (void)clearReportedWindowProxyAccessDomains
     [_windowProxyAccessDomains removeAllObjects];
 }
 
+- (void)getWebCryptoMasterKey:(void (^)(NSData *))completionHandler
+{
+    // Not so random key
+    constexpr size_t keyLength = 16;
+    uint8_t keyBytes[16] = { 0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 0xa, 0xb, 0xc, 0xd, 0xe, 0xf };
+    completionHandler([NSData dataWithBytes:keyBytes length:(keyLength)]);
+}
+
 - (NSData *)webCryptoMasterKey
```

Comment:
There shouldn't be a reason to leave this version.  It should be never called, and support for the synchronous selector is on its way to being removed.